tsconfig.json

```diff
@@ -41,7 +41,7 @@
     "server/**/*",
     "client/**/*",
     "shared/**/*",
-    "pages/**/*"
+    "pages/**/*",
```

Comment:
Oops, overlooked this earlier... JSON should not have dangling commas.

Reply:
fun fact, tsconfig jsons are not "real" json and can have comments, dangling commas, etc: https://github.com/microsoft/TypeScript/issues/20384#issuecomment-348552936

Reply:
Ahh, interesting... wasn't aware of that. I do wish they used a `jsonc` file extension or something to indicate this difference. In that case, I guess it doesn't really matter. 

For the mere purpose of consistency throughout the file, I'd still remove it. Though no longer a required change. 👍🏼